rust/src/tests/errors/etxtbsy.rs

```diff
@@ -0,0 +1,68 @@
+/// Create a test case which asserts that it returns ETXTBSY
+/// when the file is a pure procedure (shared text) file that is being executed.
+/// There are multiple forms for this macro:
+///
+/// - A basic form which takes the syscall, and optionally a `~path` argument
+///   to indicate where the `path` argument should be substituted if the path
+///   is not the only argument taken by the syscall.
+///
+/// ```
+/// // `unlink` accepts only a path as argument.
+/// enotdir_comp_test_case!(unlink);
```

Comment:
This looks copy/pasted.  You need to fix all of the identifier names in the example.